sos/policies/distros/redhat.py

```diff
@@ -138,9 +138,8 @@ def check_usrmove(self, pkgs):
         """
         if 'filesystem' not in pkgs:
             return os.path.islink('/bin') and os.path.islink('/sbin')
-        else:
-            filesys_version = pkgs['filesystem']['version']
-            return True if filesys_version[0] == '3' else False
+        filesys_version = pkgs['filesystem']['version']
+        return True if filesys_version[0] == '3' else False
```

Comment:
Isn't `return (filesys_version[0] == '3')` more concise and readable / understandable?

Anyway, that is a change different than fixinf the pylint issue, ACK to the current change either way.

Reply:
yeah, fixed that now.